backends/xnnpack/partition/configs.py

```diff
@@ -130,8 +131,11 @@
     torch.nn.functional.conv1d,
     torch.ao.nn.quantized.reference.modules.conv.Conv1d,
     torch.nn.Conv2d,
+    torch.nn.ConvTranspose2d,
```

Comment:
Should we partition these, if we don't have quantizer support yet? 
Perhaps comment these out and enable this after we add quantizer support?